sys-auth/libfprint/libfprint-1.90.7.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson udev
+
+DESCRIPTION="library to add support for consumer fingerprint readers"
+HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/freedesktop/libfprint https://gitlab.freedesktop.org/libfprint/libfprint"
+SRC_URI="https://github.com/freedesktop/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="examples +introspection"
+
+RDEPEND="dev-libs/glib:2
+	>=dev-libs/libgusb-0.3.1
+	dev-libs/nss
+	virtual/libusb:1=
+	x11-libs/gtk+:3
+	x11-libs/pixman
+	x11-libs/libX11
+	x11-libs/libXv
+	!>=${CATEGORY}/${PN}-1.90:0"
```

Comment:
Can you tell me what this does? Wouldn't there be an easier way to write this?

Reply:
It was already on it, I haven't simply touched it

Reply:
@juippis I think we can remove it, as there isn't any libfprint-1.90 on slot 0

Reply:
Hmm actually this change was done 4 months ago. I thought it'd be a lot older. Still pondering should we change it or not.

When it comes to arguments like "it was in the old ebuild", you do know we could automate all ebuilds bumps by copypasting the old ebuild until it breaks, but we prefer to have some quality included. You should check upstream build system, all configure options, what has changed between releases.

Reply:
@juippis you are right, I'm sorry for my lightness

Reply:
Keep the blocker for now, it's to prevent on breakage for systems that update very rarely. Although I don't believe it'd hit many people, we can wait for few more months before removing it.

Reply:
Ok, I will maintain it